### PR TITLE
WRO-13687 Replace external images used for Image/VirtualGridList with local svg

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Replace external images used for Image/VirtualGridList with local svg

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Implemented local svg for Image, VirtualGridList and for qa/VirtualGridList

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-13687

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: samples/sampler/stories/default/VirtualGridList.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import ri from '@enact/ui/resolution';
 import {ScrollableBase as UiScrollableBase} from '@enact/ui/Scrollable';
 import {VirtualListBase as UiVirtualListBase} from '@enact/ui/VirtualList';
 
+import {svgGenerator} from "../helper/svg";
+
+
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/VirtualGridList.js]: remove this empty line

File: samples/sampler/stories/qa/VirtualGridList.js [hunk 2]
```diff
@@ -52,7 +54,7 @@ const updateDataSize = (dataSize) => {
 			text = `Item ${count}`,
 			subText = `SubItem ${count}`,
 			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
-			source = `http://via.placeholder.com/300x300/${color}/ffffff/png?text=Image+${i}`;
+			source = svgGenerator(600, 600, color, 'ffffff', `Image ${i}`);
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/qa/VirtualGridList.js]:
```suggestion
			source = svgGenerator(300, 300, color, 'ffffff', `Image ${i}`);
```

File: samples/sampler/stories/default/VirtualGridList.js [hunk 3]
```diff
@@ -49,7 +52,11 @@ const updateDataSize = (dataSize) => {
 		const
 			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
 			count = (headingZeros + i).slice(-itemNumberDigits),
-			source = `http://via.placeholder.com/300x300/${color}/ffffff/png?text=Image+${i}`,
+			source = {
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/VirtualGridList.js]: i think here you can use only the 300 size

**Comment by daniel-stoian-lgp** [comment]
can you also update the other samples which use remote images?
Take this PR as reference: https://github.com/enactjs/sandstone/pull/1312

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: samples/perf-virtualgridlist-native/src/App.js [hunk 4]
```diff
@@ -8,13 +8,19 @@ import css from './App.module.less';
...
+);
+
 for (let i = 0; i < 1000; i++) {
 	let color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
 		count = ('00' + i).slice(-3);
 	items.push({
 		text: 'Item ' + count,
 		subText: 'SubItem ' + count,
-		url: 'http://via.placeholder.com/193x150/' + color + '/ffffff/png?text=Image ' + i
+		url: svgGenerator(300, 300, color, 'ffffff', `Image ${i}`)
```
Comment (juwonjeong) [inline samples/perf-virtualgridlist-native/src/App.js:23]: svgGenerator(193, 150, color, 'ffffff', `Image ${i}`)
And also please fix perf-virtualgridlist-translate.

Comment (juwonjeong) [inline samples/perf-virtualgridlist-native/src/App.js]:
```suggestion
		url: svgGenerator(193, 150, color, 'ffffff', `Image ${i}`)
```
And also please fix perf-virtualgridlist-translate

File: samples/sampler/stories/default/Image.js [hunk 5]
```diff
@@ -3,10 +3,12 @@ import {mergeComponentMetadata} from '@enact/storybook-utils';
...
 import {object, select} from '@enact/storybook-utils/addons/controls';
 
+import {svgGenerator} from '../helper/svg';
+
 const src = {
-	'hd':  'http://via.placeholder.com/200x200',
-	'fhd': 'http://via.placeholder.com/400x400',
-	'uhd': 'http://via.placeholder.com/600x600'
+	hd: svgGenerator(200, 200, '7ed31d', 'ffffff', '200 X 200'),
+	fhd: svgGenerator(300, 300, '7ed31d', 'ffffff', '300 X 300'),
```
Comment (juwonjeong) [inline samples/sampler/stories/default/Image.js]:
```suggestion
	fhd: svgGenerator(400, 400, '7ed31d', 'ffffff', '300 X 300'),
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**